docs/swagger.yaml

```diff
@@ -127,23 +119,45 @@ components:
           title: Validation error
           detail: Supplied landing page token is not valid
           instance: 3cff3564-4a14-4481-8dbb-431087753eed
-        example-3:
-          title: Validation error
-          detail: Merchant supplied id is not valid
-          instance: 3cff3564-4a14-4481-8dbb-431087753eed
+        Validation error:
+          type: 'https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1'
+          title: One or more validation errors occurred.
+          detail: BadRequest
+          instance: 00-616e415f5af7584eb4d9369d0fe8d7cc-48b90041faff204c-00
+          invalidParams:
+            - name: RedirectUrl
+              reason: 'The field RedirectUrl must match the regular expression ''^https://[\w\.]+([\w#!:.?+=&%@\-/]+)?$''.'
       properties:
+        type:
+          type: string
+          minLength: 1
         title:
           type: string
-          description: Title for the error
-          example: Validation error
+          minLength: 1
         detail:
           type: string
-          description: Detailed description of the error
-          example: Supplied landing page token is not valid
+          minLength: 1
         instance:
           type: string
-          description: UUID for the request
-          example: 3cff3564-4a14-4481-8dbb-431087753eed
+          minLength: 1
+        invalidParams:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
```

Comment:
All of these properties should ideally have an example. We can clean up tomorrow